tox.ini

```diff
@@ -21,10 +21,10 @@ setenv =
        PYTHONDONTWRITEBYTECODE=1
        PYTHONWARNINGS=once
 deps =
-        django22: Django>=2.2,<3.0
-        django31: Django>=3.1,<3.2
-        django32: Django>=3.2,<4.0
-        django40: Django>=4.0,<5.0
```

Comment:
I don't think we really want to change these.
(Or at least we could consider if we do or not independently of the rest of this PR)

Reply:
Actually I've reported the minimum versions of Django for each major version that are compatible with the python/django matrix (e.g. Django 3.2 is compatible with Python 3.10 starting from 3.2.9).

If you think it's an added value to the `tox.ini` file I can open a separate PR.

Reply:
Ah, I *think* I see. I wasn't able to find out what `~=` meant.
In any case, in practice the existing version is absolutely fine.